bedrock/stories/templates/stories/stories-base-article.html

```diff
@@ -0,0 +1,136 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% extends "base-protocol.html" %}
```

Comment:
**issue (not blocking but high priority):**  I see Stories- Mozilla as page title on individual article pages and I think we want specific article titles, similarly, I wonder if we need more [meta data blocks ](https://github.com/mozilla/bedrock/blob/main/bedrock/base/templates/base-protocol.html#L26) to create the expected link previews when shared

Reply:
Good call out! I'll add more meta data as I get them!